nms/app/packages/magmalte/scripts/setPassword.js

```diff
@@ -50,6 +50,7 @@ async function createUser(userObject: UserObject) {
     password: passwordHash,
     role,
     networkIDs: [],
+    tabs: [],
```

Comment:
Can we set the default tab with 'nms'. Since currently the code is primarily driven by magma usecase, no point in setting it empty. Also does this work with mariadb ? How did this work earlier ?

Reply:
So I looked into this a bit more - it's defaulting to the tabs of the organization here. This also does work with mariadb. Prior to this change, in MariaDB, the 'tabs' value would be NULL, and the user would get redirected to the tabs specified in the organization